src/styles/style.css

```diff
@@ -112,24 +97,6 @@ body {
     opacity: 0.8;
 }
 
-.one-click-app-card > div {
```

Comment:
Aren't removing these result in a different UI look compared to the existing look?

This ^ applies to all the changes in this file.